api/ticket/timeaccounting.rst

```diff
@@ -0,0 +1,157 @@
+Timeaccounting
```

Comment:
It should probably be "Time accounting", or even better "Time Accounting" because this is a heading and thus we should use [title case](https://en.wikipedia.org/wiki/Title_case).